### ISSUE TITLE
Pin header strip alignment

### ISSUE THREAD

**Issue by solarmon** [c0pperdragon/Amiga-Digital-Video#39]
Hi,
I just built my first RGBtoHDMI v2 board but having some trouble with the pin header strip alignment to the Denise socket on the motherboard (of an rev 8A A500plus).

Using a bare board you can see how the holes line up with the Denise socket holes.

![image](https://user-images.githubusercontent.com/46369787/110086356-2da9e580-7d8a-11eb-86a5-013f9f473a12.png)

I'm using the pin headers from:

https://lcsc.com/product-detail/Pin-Header-Female-Header_BOOMELE-Boom-Precision-Elec-C9742_C9742.html?fbclid=IwAR2bzglgs2G66_9C56nVjPR7m6qyY_YiOLFeSNOrObG6v17rgG3hsfyci2M

Which are different from the longer and thinner ones like at:

https://www.ebay.co.uk/itm/Uk-2-Gold-Plated-2-54mm-40-Pin-Single-Row-Straight-Round-Pin-Header-Strips/324430051449?hash=item4b8989b879:g:hWEAAOSwglZf6ybg

When I use the pin headers from LCSC, they do not seem to be line up with the Denise socket holes and I cannot get it to fit in to the socket:

![image](https://user-images.githubusercontent.com/46369787/110086384-38fd1100-7d8a-11eb-9f88-6a096d4e6e86.png)

Can somebody offer some advice on what header pins should be used and if there is any special build instructions for them.

Thank you.

**Comment by kipper2k** [comment]
for the header pins to connect to Denise socket you should use something like these...

 search in ebay "5Pcs Gold Plated 2.54mm Male 40 Pin Single Row Straight Round Pin Header Strip". Look for the best prices.  I set the vias at 0.025" drill and 0.050" diameter, this allows traces to be fed through the pind with lots of room

**Comment by solarmon** [comment]
@kipper2k 

Yes, I found those types of headers, as per my eBay link. I actually found some I have purchased previously, so I will try a build with those.

The short thicker round pin headers just don't have enough give, but it may still be possible if it was angled slightly during the build.

**Comment by solarmon** [comment]
I tried with the longer/thinner pin headers and they definitely fit better!

![image](https://user-images.githubusercontent.com/46369787/110204375-8cdf2700-7e6a-11eb-9e75-e6398845345e.png)

Oh well, lesson learnt. Those other pins are just too thick and doesn't have much give.

**Comment by kipper2k** [comment]
The  longer thiner ones are better as they dont stretch the sockets too bad and for the end user who puts the socket in, it is less likely to be removed too often and they are easier to replace if broken
